apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -1,146 +1,160 @@
-import { IconButton, Select, Theme, Tooltip } from '@material-ui/core';
-import { withStyles } from '@material-ui/core/styles';
-import { ClassNameMap, Styles } from '@material-ui/core/styles/withStyles';
-import { ArrowBack, MoreVert, Refresh } from '@material-ui/icons';
-import { Checkbox, FormControl, ListItemText, MenuItem } from '@material-ui/core';
-import { ChangeEvent, PureComponent } from 'react';
-import RightPaneStore from '../RightPaneStore';
-import { isDarkMode } from '$lib/helpers';
+import { useCallback, useEffect, useState } from 'react';
+import {
+    Box,
+    Checkbox,
+    FormControl,
+    IconButton,
+    ListItemText,
+    MenuItem,
+    Select,
+    Tooltip,
+    type SelectChangeEvent,
+    type SxProps,
+} from '@mui/material';
+import { ArrowBack, Visibility, Refresh } from '@mui/icons-material';
+import RightPaneStore, { type SectionTableColumn } from '../RightPaneStore';
 
-const styles: Styles<Theme, object> = {
-    buttonRow: {
-        width: '100%',
-        zIndex: 3,
-        marginBottom: 8,
-        position: 'absolute',
-        pointerEvents: 'none',
-    },
-    button: {
-        backgroundColor: 'rgba(236, 236, 236, 1)',
-        marginRight: 5,
-        boxShadow: '2',
-        color: 'black',
-        '&:hover': {
-            backgroundColor: 'grey',
-        },
-        pointerEvents: 'auto',
+/**
+ * All the interactive buttons have the same styles.
+ */
+const buttonSx: SxProps = {
+    backgroundColor: 'rgba(236, 236, 236, 1)',
+    marginRight: 5,
```

Comment:
```suggestion
    marginRight: 1,
    padding: 1.5,
```

Corrects the sizing and margins.